src/components/Navbar.tsx

```diff
@@ -8,17 +14,13 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink to="/" className={selectedPage}>
             Home
-          </a>
+          </NavLink>
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
+          <NavLink aria-current="page" className={selectedPage} to="/people">
```

Comment:
The 'aria-current' attribute is used to indicate the current page within a set of navigation links. It should only be used on the link that represents the current page. Consider dynamically setting this attribute based on the current route.